src/App.js

```diff
@@ -1,24 +1,46 @@
-import logo from './logo.svg';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import { ToastContainer } from 'react-toastify';
+import Navbar from './components/navbar';
+import Home from './components/home';
+import Houses from './components/houses';
+import HouseDetails from './components/houses/houseDetails';
+import Reservations from './components/reservations';
+import Login from './components/auth/login';
+import Register from './components/auth/register';
+import PrivateRoute from './components/auth/private-route';
+import AddHouse from './components/houses/add-house';
+import DeleteHouses from './components/deleteHouses';
+import ReserveHouse from './components/reservations/reserve-house';
+import 'react-toastify/dist/ReactToastify.css';
 import './App.css';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 8):

![image](https://github.com/ahmedeid6842/book_an_appointment_frontend/assets/57421823/1d6ef1a0-acb2-47a3-bbe1-a51984c522ce)


That way, your code will meet the latest standards. 💪